spark/src/main/java/org/apache/iceberg/spark/source/Writer.java

```diff
@@ -108,6 +109,7 @@
     this.replacePartitions = replacePartitions;
     this.applicationId = applicationId;
     this.wapId = wapId;
+    this.isWapEnabled = isWapTable();
```

Comment:
Can we roll back the changes to this file and include them in a separate commit? There's no need to make changes here that could be done in the other planned update.